inc/functions/files.php

```diff
@@ -99,6 +99,17 @@ function get_rocket_config_file() {
 		$buffer .= '$rocket_common_cache_logged_users = 1;' . "\n";
 	}
 
+	/**
+	 * Filters the use of the mobile cache version for tablets
+	 * 0 will serve desktop to tablets, 1 will serve mobile to tablets
+	 *
+	 * @since 3.2
+	 * @author Remy Perona
+	 *
+	 * @param int $mobile_for_tablet valid values are 0 or 1.
+	 */
+	$buffer .= '$rocket_cache_mobile_files_tablet = ' . apply_filters( 'rocket_cache_mobile_files_tablet', 0 ) . ';' . "\n";
```

Comment:
Can't we use something like this instead?

```php
$rocket_cache_tablet = 'mobile';
$rocket_cache_tablet = 'desktop';
```

I think it can be understood more easily.